From 4be5f6686f4777466ec8dd7f585afec0c4d65b5b Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Thu, 8 Mar 2018 22:46:55 +0100
Subject: [PATCH 59/74] fm: Resolve required dependency qcom.fmradio.xml
 linkage

 * LOCAL_REQUIRED_MODULES should be used instead of
    LOCAL_ADDITIONAL_DEPENDENCIES to link a module

 * fm/Android.mk:
    error: qcom.fmradio: LOCAL_ADDITIONAL_DEPENDENCIES
    must only contain paths (not module names)

Change-Id: Ic0bebf752101f5d16abc52e085bae4acd0b0c61c
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 65587c8..700ec4f 100644
--- a/Android.mk
+++ b/Android.mk
@@ -12,7 +12,7 @@ LOCAL_JNI_SHARED_LIBRARIES := libqcomfm_jni
 LOCAL_SHARED_LIBRARIES := liblog
 LOCAL_MODULE:= qcom.fmradio
 
-LOCAL_ADDITIONAL_DEPENDENCIES := qcom.fmradio.xml
+LOCAL_REQUIRED_MODULES := qcom.fmradio.xml
 
 include $(BUILD_JAVA_LIBRARY)
 
-- 
2.17.1

